app/models/publish_target.rb

```diff
@@ -10,12 +10,21 @@ def as_json(_options = {})
       string_key: string_key,
       display_label: display_label,
       publish_url: publish_url,
-      api_key: api_key
+      api_key: api_key,
+      is_allowed_doi_target: is_allowed_doi_target,
+      doi_priority: doi_priority
     }
   end
 
-  # TODO: Remove when DOI service is implemented
+  def from_json(json, include_root = include_root_in_json)
+    hash = ActiveSupport::JSON.decode(json)
```

Comment:
I usually use ruby's built-in `JSON.parse`.  Any additional advantage to using `ActiveSupport::JSON`?

Reply:
Nope, I'll remove it.